apps/protocol/lib/lexical/protocol/convertibles/lexical.document.range.ex

```diff
@@ -2,8 +2,8 @@ defimpl Lexical.Convertible, for: Lexical.Document.Range do
   alias Lexical.Document
   alias Lexical.Protocol.Conversions
 
-  def to_lsp(%Document.Range{} = position, context_document) do
-    Conversions.to_lsp(position, context_document)
+  def to_lsp(%Document.Range{} = position) do
+    Conversions.to_lsp(position)
```

Comment:
This is a typo that was already there but we should just fix it: position -> range (looks like same typo is in the clause below as well)